pkg/enqueue-bundle/DependencyInjection/Configuration.php

```diff
@@ -94,6 +94,9 @@ private function getJobConfiguration(): ArrayNodeDefinition
         }
 
         return (new ArrayNodeDefinition('job'))
+            ->children()
+                ->booleanNode('default_mapping')->defaultTrue()->end()
+            ->end()
```

Comment:
Actually we can use this place to add information about what this property does:

```diff
            ->children()
-                ->booleanNode('default_mapping')->defaultTrue()->end()
+                ->booleanNode('default_mapping')
+                    ->defaultTrue()
+                    ->info('Adds bundle's default Job entity mapping to application's entity manager')
+                ->end()
            ->end()
```